kolibri/content/models.py

```diff
@@ -170,11 +177,14 @@ def save(self, *args, **kwargs):
             md5 = hashlib.md5()
             for chunk in self.content_copy.chunks():
                 md5.update(chunk)
-
             self.checksum = md5.hexdigest()
             self.available = True
             self.file_size = self.content_copy.size
-            self.extension = os.path.splitext(self.content_copy.name)[1]
+            # if feed a format that doesn't exist yet will break the app
+            # not sure how we want to handle this yet
```

Comment:
yes, do we have a proper way to send error from backend to front-end yet? I think we have something like this in kalite.